src/examples/payment/createIndustryData.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * Mercado Pago Payment Capture.
```

Comment:
[req] Alterar comentário
```suggestion
 * Mercado Pago Payment with Industry Data.
```